experimental/BasisLieHighestWeight/src/MainAlgorithm.jl

```diff
@@ -85,7 +96,18 @@ function basis_lie_highest_weight_compute(
   set_attribute!(
     mb, :algorithm => basis_lie_highest_weight_compute, :minkowski_gens => minkowski_gens
   )
-  return mb
+
+  if !isnothing(vertices_of_polytope)
+    set_attribute!(
+      mb,
+      :volume_of_polytope => volume(convex_hull(vertices_of_polytope[highest_weight(V)])),
+    )
+    set_attribute!(
+      mb,
+      :polytope => convex_hull(vertices_of_polytope[highest_weight(V)]),
+    )
```

Comment:
```suggestion
    P = convex_hull(vertices_of_polytope[highest_weight(V)])
    set_attribute!(
      mb,
      :polytope => P,
      :volume_of_polytope => volume(P),
    )
```
you probably don't want to compute the convex hull twice